projects/addon-mobile/components/mobile-calendar/mobile-calendar.component.ts

```diff
@@ -145,6 +165,23 @@ export class TuiMobileCalendarComponent implements AfterViewInit {
         valueChanges.pipe(takeUntil(this.destroy$)).subscribe(value => {
             this.value = value;
         });
+
+        this.value$
+            .pipe(
+                skip(1),
+                distinctUntilChanged((a, b) => a?.toString() === b?.toString()),
+                takeUntil(this.destroy$),
+            )
+            .subscribe(this.valueChange);
```

Comment:
Instead of subscribing, I think it's better to do `readonly valueChange = this.value$.pipe(...)`